source/stylesheets/site.css.scss

```diff
@@ -133,11 +133,9 @@ aside {
     height: 100%;
     overflow-y: auto;
 
-    h1 {
-      margin: 0;
-    }
-
-    h1 a {
+    & > a {
```

Comment:
is `&` an SCSS thing, or a CSS thing?

Reply:
It is a SCSS thing to say "Put the parent selector here".